runtime/near-vm/lib/vm/src/instance/mod.rs

```diff
@@ -349,12 +349,12 @@ impl Instance {
     }
 
     /// Return a pointer to initial stack limit.
-    pub fn stack_limit_initial_ptr(&self) -> *mut i32 {
+    pub fn stack_limit_initial_ptr(&self) -> *mut u32 {
```

Comment:
Isn't this the secondary stack limiter thing? We should get rid of it.

Reply:
It originally was, but I removed the secondary stack limiter instrumentation, and instead repurposed the memory location in order to use it for the now-main stack limiter, as it needs basically the same mechanism of resetting the stack limit upon a new contract execution and being easy to access from wasm.